src/InterpreterStatements.ml

```diff
@@ -832,8 +832,9 @@ let rec eval_statement (config : C.config) (st : A.statement) : st_cm_fun =
         (* Compose and apply *)
         comp cf_eval_rvalue cf_assign cf ctx
     | A.AssignGlobal { dst; global } ->
+        (* What codegen do we want ? *)
```

Comment:
In the symbolic case, the cleanest way is to add a variant `ReadGlobal of GlobalDeclId.Id * symbolic_value` in `SymbolicAst.expression`. The symbolic value would be freshly generated upon reading the global.